test/langtools/tools/javac/Diagnostics/8295024/T8295024.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * @test /nodynamiccopyright/
+ * @bug     8295024
+ * @summary Cyclic constructor error is non-deterministic and inconsistent
+ * @compile T8295024.java
```

Comment:
both the `@compile` line and the `@run` one can be removed, jtreg will do by default what you expect